### DISCUSSION TITLE
Revert all settings on Windows 11

### DISCUSSION THREAD

**Discussion by Tech3963** [Open-Shell/Open-Shell-Menu#1243] [Q&A]
So...I decided Open-shell isn't for me. I uninstalled, and even deleted all registry entries I could find, but my right-click context menu won't switch back to the Windows 11 style. How can I restore the Windows 11 right-click menu?

**Comment by bonzibudd** [comment]
Open-Shell shouldn't have any effect on the right-click menu in Windows 11. Are you using ExplorerPatcher? Also if you could attach a screenshot that would be useful.
